lib/vty.c

```diff
@@ -2631,7 +2726,7 @@ bool vty_read_config(struct nb_config *config, const char *config_file,
 		if (strstr(config_default_dir, "vtysh") == NULL) {
 			ret = stat(integrate_default, &conf_stat);
 			if (ret >= 0) {
-				read_success = true;
+				// read_success = true;
```

Comment:
Any reason to keep this, the variable is gone.

Reply:
removed commented out code.